superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigModal.tsx

```diff
@@ -369,11 +386,34 @@ export function FiltersConfigModal({
     setOrderedFilters(newOrderedFilter);
   };
 
+  const buildDependencyMap = useMemo(
```

Comment:
If you use `useCallback`, you can get rid of one of `() =>`
Like
```
const buildDependencyMap = useCallback(() => {
   ...
})
```

Reply:
But I also want to cache the result and not just the function.